smithy-model/src/main/java/software/amazon/smithy/model/validation/validators/ExamplesTraitValidator.java

```diff
@@ -53,23 +53,34 @@ private List<ValidationEvent> validateExamples(Model model, OperationShape shape
                 events.addAll(input.accept(validator));
             });
 
-            model.getShape(shape.getOutputShape()).ifPresent(output -> {
-                NodeValidationVisitor validator = createVisitor(
-                        "output", example.getOutput(), model, shape, example);
-                events.addAll(output.accept(validator));
-            });
+            boolean isOutputDefined = example.getOutput().isPresent();
+            boolean isErrorDefined = example.getError().isPresent();
+
+            if (isOutputDefined && isErrorDefined) {
```

Comment:
If this is the behaviour we want, this needs to be mentioned in the doc as well.
https://smithy.io/2.0/spec/documentation-traits.html?highlight=example#examples-trait

Reply:
Great point!